src/redux/missions/missionsSlice.js

```diff
@@ -1,13 +1,11 @@
-// // src/redux/books/booksSlice.js
-
 import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
 import axios from 'axios';
 
 const initialState = {
   missions: [],
   isLoading: false,
   error: undefined,
-  missionJoined: true,
```

Comment:
Kindly consider setting the initial value for `missionJoined` to be `false`, maybe this is the reason I cannot find "Join Mission" button.